e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-enabled/gef-deal-add-bank-review-date.spec.js

```diff
@@ -0,0 +1,169 @@
+      });
+    });
+
+    describe('when the bank review date is valid and only the cover end date is being changed', () => {
+      beforeEach(() => {
+        amendmentsPage.navigateToIsUsingFacilityEndDatePage();
+        amendmentsPage.continueAmendment().click();
+
+        cy.url().should('contain', 'bank-review-date');
+        amendmentsPage.amendmentBankReviewDateDayInput().clear().type(dateConstants.todayDay);
+        amendmentsPage.amendmentBankReviewDateMonthInput().clear().type(dateConstants.todayMonth);
+        amendmentsPage.amendmentBankReviewDateYearInput().clear().type(dateConstants.todayYear);
+        amendmentsPage.continueAmendment().click();
+      });
+
+      it('should continue to "Check your answers" page', () => {
+        cy.url().should('contain', 'check-answers');
+        amendmentsPage.amendmentAnswerIsUsingFacilityEndDate().should('have.text', 'No');
+        amendmentsPage.amendmentAnswerBankReviewDate().should('have.text', dateConstants.todayFullString);
+        amendmentsPage.amendmentAnswerFacilityEndDate().should('not.exist');
+      });
+    });
+
+    describe('when on the check your answers page', () => {
+      beforeEach(() => {
+        amendmentsPage.navigateToIsUsingFacilityEndDatePage();
+        amendmentsPage.continueAmendment().click();
+
+        cy.url().should('contain', 'bank-review-date');
+        amendmentsPage.continueAmendment().click();
+
+        cy.url().should('contain', 'check-answers');
+      });
+
+      it('should allow bank review date amendments on the `Check your answers` page', () => {
+        amendmentsPage.amendmentAnswerBankReviewDateChangeLink().click();
+
+        cy.url().should('contain', 'bank-review-date');
+        amendmentsPage.amendmentBankReviewDateDayInput().clear().type(dateConstants.threeMonthsOneDayDay);
+        amendmentsPage.amendmentBankReviewDateMonthInput().clear().type(dateConstants.threeMonthsOneDayMonth);
+        amendmentsPage.amendmentBankReviewDateYearInput().clear().type(dateConstants.threeMonthsOneDayYear);
+        amendmentsPage.continueAmendment().click();
+
+        cy.url().should('contain', 'check-answers');
+        amendmentsPage.amendmentAnswerIsUsingFacilityEndDate().should('have.text', 'No');
+        amendmentsPage.amendmentAnswerBankReviewDate().should('have.text', dateConstants.threeMonthsOneDayFullString);
+        amendmentsPage.amendmentAnswerFacilityEndDate().should('not.exist');
+      });
+
+      describe('when submitting and visiting the facility summary page', () => {
+        beforeEach(() => {
+          amendmentsPage.continueAmendment().click();
+          cy.visit(relative(`/case/${dealId}/facility/${facility._id}`));
+        });
+
+        it('should correctly display amended values', () => {
+          facilityPage.facilityIsUsingFacilityEndDate().should('have.text', 'No');
+          facilityPage.facilityBankReviewDate().should('have.text', dateConstants.threeMonthsOneDayFullMonthString);
```

Comment:
Can we update these `should('have.text')` tests to use text/trim and equals please? Seems to be a few instances added in this PR; I think `have.text` is only a partial match.

e.g

```js
facilityPage.facilityBankReviewDate()
  .invoke('text')
  .then((text) => {
    expect(text.trim()).to.equal(dateConstants.threeMonthsOneDayFullMonthString);
  });
```

This'll also make it easier to update [this assertText PR](https://github.com/UK-Export-Finance/dtfs2/pull/3407) when merging the latest from `main` 🙏

Reply:
Actually sorry @AlexBramhill @BethThomas141  - don't worry about this, there are 142 instances of `.should('have.text'`, can replace all in one go as part of DTFS2-7320 or an iteration of DTFS2-7320